vignettes/deployment.Rmd

```diff
@@ -25,27 +25,14 @@ The Cloud ML prediction service makes use of models exported through the
 `export_savedmodel()` function which is available for models created using the [tensorflow](https://tensorflow.rstudio.com/tensorflow/), [keras](https://tensorflow.rstudio.com/keras/) and
 [tfestimators](https://tensorflow.rstudio.com/tfestimators/) packages or any other tool that support the [tf.train.Saver](https://www.tensorflow.org/api_docs/python/tf/train/Saver) interface.
 
-For instance, we can define and export a simple model by running:
+For instance, we can use `examples/keras/train.R` included in this package to define
+and train an MNIST keras model by running:
 
 ```{r}
-library(tensorflow)
-library(cloudml)
-
-# define output folder
-model_dir <- tempfile()
-
-# define simple string-based tensor operations
-sess <- tf$Session()
-name <- tf$placeholder(tf$string)
-hello <- tf$string_join(inputs = c("Hello, ", name, "!"))
-
-# export as a savedmodel
-export_savedmodel(
-  sess,
-  model_dir,
-  inputs = list(name = name),
-  outputs = list(hello = hello)
-)
+library(tfruns)
+
+training_file <- system.file("examples/keras/train.R", package = "cloudml")
+training_run(training_file)
```

Comment:
I don't love introducing tfruns and sourcing the example here. I would almost rather have the code inline.

Reply:
Also, I don't think training_run is even necessary for this case (source would be sufficient)

Reply:
Sounds good, will add code inline.